docs/contributing/contributing-to-airbyte/developing-locally.md

```diff
@@ -29,8 +29,8 @@ This will build all the code and run all the unit tests.
 ## Run in `dev` mode with `docker-compose`
 
 ```bash
-./gradlew build
-docker-compose --env-file .env.dev -f docker-compose.yaml -f docker-compose.dev.yaml up
+./gradlew build composeBuild
```

Comment:
this file has the only relevant changes

Reply:
composeBuild is included in build already